src/postgres_mcp/dta/dta_calc.py

```diff
@@ -289,9 +291,8 @@ async def analyze_workload(
         query_list: list[str] | None = None,
         min_calls: int = 50,
         min_avg_time_ms: float = 5.0,
-        limit: int = 100,
+        limit: int = MAX_QUERIES_LIMIT,
```

Comment:
can we name this argument more descriptively?

Reply:
Done.